### DISCUSSION TITLE
Laser Image Preparation Website

### DISCUSSION THREAD

**Discussion by Tribo-2dot0** [arkypita/LaserGRBL#1460] [Ideas]
I didn't know, there are some good tools ... I thought it was really cool.

https://www.imag-r.com/

**Comment by StuartB4** [comment]
Yeah, just remember to use Passthrough  in LaserGRBL if you process your images there.

**Comment by StuartB4** [comment]
This is from the LaserGRBL Facebook group page. 

`With v4.1.1 a new Passthrough option was added.
With this option the image is processed using the dimensions and resolution set in the file, without further modifications and resizing by LaserGRBL.
This option is very useful for those who use an external program to correct brightness and contrast, or to do dithering, and do not want the image to be processed a second time.`
